### PR TITLE
feat(account-lib): skeleton code for avalanche c-chain in account-lib

### PR DESCRIPTION
**Pull request by shiksb**
Ticket: STLX-3250

### CODE REVIEW

File: modules/account-lib/src/coin/avaxc/transferBuilder.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+/** AVAXC transfer builder */
+export class TransferBuilder extends EthTransferBuilder {
+  /**
+   * Get the prefix used in generating an operation hash for sending tokens
+   *
+   * @returns the string prefix
+   */
+  protected getTokenOperationHashPrefix(): string {
+    return 'AVAXC-ERC20';
```
Comment (smaria95) [inline modules/account-lib/src/coin/avaxc/transferBuilder.ts:11]: @CapnMigraine 
This string should match the one on `SendMultisigToken` method on the contract (which is `AVAX-ERC20` at the moment).
Should we update the contract PR or this PR instead?

File: modules/account-lib/src/coin/avaxc/transferBuilder.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  }
+
+  /**
+   * Get the prefix used in generating an operation hash for sending native coins
+   *
+   * @returns the string prefix
+   */
+  protected getNativeOperationHashPrefix(): string {
+    return 'AVAXC';
```
Comment (smaria95) [inline modules/account-lib/src/coin/avaxc/transferBuilder.ts:20]: @CapnMigraine 
The same here but for contract method `SendMultisig`

File: modules/account-lib/src/coin/avaxc/transferBuilder.ts (hunk 1 above)
Comment (CapnMigraine) [reply modules/account-lib/src/coin/avaxc/transferBuilder.ts:11]: @shiksb I think we can assume that if we were to support an asset like `AVAX`, but with multiple eth-like chains, the asset should be the same across them —i.e. `AVAX` on the X/P chains—, and the contract would be the same too. We can use `AVAX-ERC20` and `AVAX`.

**Review by CapnMigraine — APPROVED** [review]
lgtm

**Pull request merged**